packages/react/src/components/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -137,35 +107,34 @@ describe('Breadcrumb', () => {
       { text: 'Test2', key: 'Test2', onClick: clickCallback },
     ];
 
-    wrapper = mount(<Breadcrumb items={items2} />);
+    const { container } = render(<Breadcrumb items={items2} />);
 
-    const renderedItems = wrapper.find('.ms-Breadcrumb-itemLink').hostNodes();
+    const renderedItems = container.querySelectorAll('.ms-Breadcrumb-itemLink');
 
-    renderedItems.at(0).simulate('click');
+    userEvent.click(renderedItems[0]);
     expect(callbackValue).toEqual('Test1');
 
-    renderedItems.at(1).simulate('click');
+    userEvent.click(renderedItems[1]);
     expect(callbackValue).toEqual('Test2');
   });
 
   it('moves items to overflow in the correct order', () => {
-    wrapper = mount(<Breadcrumb items={items} maxDisplayedItems={2} />);
+    const { container } = render(<Breadcrumb items={items} maxDisplayedItems={2} />);
 
-    expect(wrapper.find('.ms-Breadcrumb-item').first().text()).toEqual('TestText3');
+    expect(container.querySelectorAll('.ms-Breadcrumb-item')[0].textContent).toEqual('TestText3');
   });
 
   it('supports native props on the root element', () => {
-    wrapper = mount(<Breadcrumb items={items} maxDisplayedItems={2} role="region" />);
+    const { getByRole } = render(<Breadcrumb items={items} maxDisplayedItems={2} role="region" />);
 
-    expect(wrapper.find('.ms-Breadcrumb').prop('role')).toEqual('region');
+    expect(getByRole('region', { hidden: true })).toBeTruthy();
```

Comment:
testing-library deemed this as an "inaccessible" role thus I set `hidden:true` to gain access to this element

Reply:
Thanks to @ecraig12345 for investigating why elements were inaccessible to begin with [here](https://github.com/microsoft/fluentui/pull/22247#discussion_r838125482), i've now changed this approach to use jest timers before trying to get the role.